Fix linting issues

Fix linting issues flagged in tests.

```
/home/runner/work/localgov_events/localgov_events/html/web/modules/contrib/localgov_events/js/localgov_events_date_picker.js
   6:2   warning  Unexpected unnamed function                   func-names
   8:29  warning  'context' is defined but never used           no-unused-vars
   8:38  warning  'settings' is defined but never used          no-unused-vars
   9:41  warning  Unexpected unnamed function                   func-names
  10:13  error    Identifier 'start_date' is not in camel case  camelcase
  11:13  error    Identifier 'end_date' is not in camel case    camelcase
  14:9   error    Expected a default case                       default-case
  16:13  error    Identifier 'start_date' is not in camel case  camelcase
  17:13  error    Identifier 'end_date' is not in camel case    camelcase
  21:13  error    Unexpected lexical declaration in case block  no-case-declarations
  22:13  error    Identifier 'start_date' is not in camel case  camelcase
  23:13  error    Identifier 'end_date' is not in camel case    camelcase
  28:13  error    Unexpected lexical declaration in case block  no-case-declarations
  30:13  error    Unexpected lexical declaration in case block  no-case-declarations
  31:13  error    Identifier 'start_date' is not in camel case  camelcase
  32:13  error    Identifier 'end_date' is not in camel case    camelcase
  36:13  error    Identifier 'start_date' is not in camel case  camelcase
  37:13  error    Identifier 'end_date' is not in camel case    camelcase
  47:15  error    Identifier 'start_date' is not in camel case  camelcase
  53:15  error    Identifier 'end_date' is not in camel case    camelcase
  60:15  error    Identifier 'start_date' is not in camel case  camelcase
  61:15  error    Identifier 'end_date' is not in camel case    camelcase
  66:13  error    Identifier 'start_date' is not in camel case  camelcase
  68:13  error    Identifier 'start_date' is not in camel case  camelcase
  73:13  error    Identifier 'end_date' is not in camel case    camelcase
  74:40  error    Identifier 'end_date' is not in camel case    camelcase

✖ 26 problems (22 errors, 4 warnings)
```

--- a/css/localgov_events.css
+++ b/css/localgov_events.css
@@ -14,12 +14,12 @@
   margin: 0 1.5rem 1rem 0;
 }
 
-form[id^=views-exposed-form-localgov-events-listing-page-all-events] input[type=submit],
-form[id^=views-exposed-form-localgov-events-search-events-page] input[type=submit] {
+form[id^="views-exposed-form-localgov-events-listing-page-all-events"] input[type="submit"],
+form[id^="views-exposed-form-localgov-events-search-events-page"] input[type="submit"] {
   margin-top: 1rem;
 }
 
-form[id^=views-exposed-form-localgov-events-listing-page-all-events] input[type=submit]:first-child,
-form[id^=views-exposed-form-localgov-events-search-events-page] input[type=submit]:first-child {
+form[id^="views-exposed-form-localgov-events-listing-page-all-events"] input[type="submit"]:first-child,
+form[id^="views-exposed-form-localgov-events-search-events-page"] input[type="submit"]:first-child {
   margin-top: 0;
 }

--- a/js/localgov_events_date_picker.js
+++ b/js/localgov_events_date_picker.js
@@ -3,75 +3,77 @@
  * Events date picker functionality.
  */
 
-(function ($) {
+(function localgovEventsDatePickerScript($) {
   Drupal.behaviors.localgovEventsDatePicker = {
-    attach: function attach(context, settings) {
+    attach: function attach() {
       $('.js-date-picker').on('change', function () {
-        let start_date = null;
-        let end_date = null;
+        let startDate = null;
+        let endDate = null;
         const today = new Date();
 
         switch (this.value) {
-          case 'today':
-            start_date = today;
-            end_date = today;
+          case 'today': {
+            startDate = today;
+            endDate = today;
             break;
+          }
 
-          case 'tomorrow':
+          case 'tomorrow': {
             const tomorrow = today.setDate(today.getDate() + 1);
-            start_date = new Date(tomorrow);
-            end_date = new Date(tomorrow);
+            startDate = new Date(tomorrow);
+            endDate = new Date(tomorrow);
             break;
+          }
 
-          case 'this_week':
+          case 'this_week': {
             // First day is the day of the month - the day of the week.
             const first = today.getDate() - today.getDay() + 1;
             // Last day is the first day + 6.
             const last = first + 6;
-            start_date = new Date(today.setDate(first));
-            end_date = new Date(today.setDate(last));
+            startDate = new Date(today.setDate(first));
+            endDate = new Date(today.setDate(last));
             break;
+          }
 
-          case 'this_month':
-            start_date = new Date(today.getFullYear(), today.getMonth(), 1, 1);
-            end_date = new Date(
-              today.getFullYear(),
-              today.getMonth() + 1,
-              0,
-              1,
-            );
+          case 'this_month': {
+            startDate = new Date(today.getFullYear(), today.getMonth(), 1, 1);
+            endDate = new Date(today.getFullYear(), today.getMonth() + 1, 0, 1);
             break;
+          }
 
-          case 'next_month':
+          case 'next_month': {
             if (today.getMonth() < 11) {
-              start_date = new Date(
+              startDate = new Date(
                 today.getFullYear(),
                 today.getMonth() + 1,
                 1,
                 1,
               );
-              end_date = new Date(
+              endDate = new Date(
                 today.getFullYear(),
                 today.getMonth() + 2,
                 0,
                 1,
               );
             } else {
-              start_date = new Date(today.getFullYear() + 1, 0, 1, 1);
-              end_date = new Date(today.getFullYear() + 1, 1, 0, 1);
+              startDate = new Date(today.getFullYear() + 1, 0, 1, 1);
+              endDate = new Date(today.getFullYear() + 1, 1, 0, 1);
             }
             break;
+          }
+          default: {
+            startDate = null;
+            endDate = null;
+          }
         }
 
-        if (start_date) {
-          $('.js-date-picker-start').val(
-            start_date.toISOString().substr(0, 10),
-          );
+        if (startDate) {
+          $('.js-date-picker-start').val(startDate.toISOString().substr(0, 10));
         } else {
           $('.js-date-picker-start').val('');
         }
-        if (end_date) {
-          $('.js-date-picker-end').val(end_date.toISOString().substr(0, 10));
+        if (endDate) {
+          $('.js-date-picker-end').val(endDate.toISOString().substr(0, 10));
         } else {
           $('.js-date-picker-end').val('');
         }

--- a/js/localgov_events_date_picker.js
+++ b/js/localgov_events_date_picker.js
@@ -6,15 +6,12 @@
 (function ($) {
   Drupal.behaviors.localgovEventsDatePicker = {
     attach: function attach(context, settings) {
-
-      $('.js-date-picker').on('change', function() {
-
+      $('.js-date-picker').on('change', function () {
         let start_date = null;
         let end_date = null;
         const today = new Date();
 
-        switch(this.value) {
-
+        switch (this.value) {
           case 'today':
             start_date = today;
             end_date = today;
@@ -36,37 +33,49 @@
             break;
 
           case 'this_month':
-            start_date = new Date(today.getFullYear(), today.getMonth() , 1, 1);
-            end_date = new Date(today.getFullYear(), today.getMonth() + 1, 0, 1);
+            start_date = new Date(today.getFullYear(), today.getMonth(), 1, 1);
+            end_date = new Date(
+              today.getFullYear(),
+              today.getMonth() + 1,
+              0,
+              1,
+            );
             break;
 
           case 'next_month':
             if (today.getMonth() < 11) {
-              start_date = new Date(today.getFullYear(), today.getMonth() + 1, 1, 1);
-              end_date = new Date(today.getFullYear(), today.getMonth() + 2, 0, 1);
-            }
-            else {
+              start_date = new Date(
+                today.getFullYear(),
+                today.getMonth() + 1,
+                1,
+                1,
+              );
+              end_date = new Date(
+                today.getFullYear(),
+                today.getMonth() + 2,
+                0,
+                1,
+              );
+            } else {
               start_date = new Date(today.getFullYear() + 1, 0, 1, 1);
               end_date = new Date(today.getFullYear() + 1, 1, 0, 1);
             }
             break;
-
         }
 
         if (start_date) {
-          $('.js-date-picker-start').val(start_date.toISOString().substr(0, 10));
-        }
-        else {
+          $('.js-date-picker-start').val(
+            start_date.toISOString().substr(0, 10),
+          );
+        } else {
           $('.js-date-picker-start').val('');
         }
         if (end_date) {
           $('.js-date-picker-end').val(end_date.toISOString().substr(0, 10));
-        }
-        else {
+        } else {
           $('.js-date-picker-end').val('');
         }
       });
-
-    }
-  }
+    },
+  };
 })(jQuery);

--- a/js/localgov_events_date_picker.js
+++ b/js/localgov_events_date_picker.js
@@ -6,12 +6,12 @@
 (function localgovEventsDatePickerScript($) {
   Drupal.behaviors.localgovEventsDatePicker = {
     attach: function attach() {
-      $('.js-date-picker').on('change', function () {
+      $('.js-date-picker').on('change', (event) => {
         let startDate = null;
         let endDate = null;
         const today = new Date();
 
-        switch (this.value) {
+        switch (event.target.value) {
           case 'today': {
             startDate = today;
             endDate = today;

